### ISSUE TITLE
Drag position not offset by scroll position

### ISSUE THREAD

**Issue by blinkingmatt** [antoniandre/splitpanes#28]

**Comment by blinkingmatt** [comment]
When the splitpanes containing element is scrolled (e.g. vertical splitpanes in horizontally scrolled element) drag (resizer) position jumps to (start drag position - scroll position).
See:
[https://codepen.io/blinkingmatt/pen/bZrymx]

**Comment by blinkingmatt** [comment]
Solution could be to adapt getCurrentMouseDrag()

```
getCurrentMouseDrag: function getCurrentMouseDrag(e) {

      // according
      // https://stackoverflow.com/questions/4962710/get-mouse-position-in-scrollable-div
      function recursive_offset (aobj) {
        var currOffset = {
          x: 0,
          y: 0
        } 
        var newOffset = {
          x: 0,
          y: 0
        }    
       
        if (aobj !== null) {
       
          if (aobj.scrollLeft) { 
            currOffset.x = aobj.scrollLeft;
          }
       
          if (aobj.scrollTop) { 
            currOffset.y = aobj.scrollTop;
          } 
       
          if (aobj.parentNode !== undefined) { 
            newOffset = recursive_offset(aobj.parentNode);   
          }
       
          currOffset.x = currOffset.x + newOffset.x;
          currOffset.y = currOffset.y + newOffset.y; 
        }
        return currOffset;
      }

      let offs = recursive_offset( this.container.vnode );

      return {
        x: 'ontouchstart' in window ? e.touches[0].clientX : e.clientX + offs.x,
        y: 'ontouchstart' in window ? e.touches[0].clientY : e.clientY + offs.y
      };
    },

```

**Comment by antoniandre** [comment]
Hi @blinkingmatt, 
Thank you for reporting, this is now fixed in version 1.14.2.
Sorry for the long wait!

**Comment by blinkingmatt** [comment]
Nice,

Regards,
Matt

Von: Antoni <notifications@github.com>
Gesendet: Dienstag, 16. April 2019 08:53
An: antoniandre/splitpanes <splitpanes@noreply.github.com>
Cc: Matthias Kirst <matthias@blink-dx.com>; Mention <mention@noreply.github.com>
Betreff: Re: [antoniandre/splitpanes] Drag position not offset by scroll position (#28)


Hi @blinkingmatt<https://github.com/blinkingmatt>,
Thank you for reporting, this is now fixed in version 1.14.2.
Sorry for the long wait!
